packages/jest-fake-chalk/src/index.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) 2014, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ * @flow
+ */
+
+import ansiStyles from 'ansi-styles';
```

Comment:
I thought about doing a `try-catch` around the normal chalk, and just create a fake on if it fails. But I don't think that matters?